portability-types-common/src/main/java/org/datatransferproject/types/common/models/mail/MailContainerModel.java

```diff
@@ -36,10 +39,18 @@ public String getId() {
     return id;
   }
 
+  @JsonIgnore(false)
+  @Override
   public String getName() {
     return name;
   }
 
+  @Nonnull
+  @Override
+  public String getIdempotentId() {
+    return getId();
+  }
```

Comment:
Good catch. Yeah, let's play it safe and file a separate bug for `MailContainerModel` and drop this file's changes from this PR.

---
note for future-selves:


`GoogleMailImporter` is a solid example of not-great javadoc (and how we actually would've liked to have good javadoc). In place of good docs, I'm looking at the code and: (a) I can't find any examples of the `id` of that code ever being utilized (even in test code); (b) I'm thinking it might just be poorly documented dead-code in which case `getName()` _is_ surprisingly the correct "idempotent ID".

_(with `GoogleMailImporter` lacking docs as motivation, maybe we should follow-up and document `ImportableItem` fields; eg: particularly to explain what the intended use of "name" is)_